students/mariusz_michalczyk/lesson_08_regex/regex_strip.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+
+def strip_mm(str, replacement=''):
+    if not replacement:
+        rgx_strip = re.compile(r'\s*(\d*\w*)\s*')
+        matched = rgx_strip.search(str)
+        return matched.group(1)
+    else:
+        regex = re.compile(r'^[{0}]*|[{0}]*$'.format(replacement))
```

Comment:
Almost, but you need to escape special characters, `re.escape(replacement)`, otherwise `.` will catch everything.